src/sentry/static/sentry/app/stores/hookStore.tsx

```diff
@@ -2,7 +2,7 @@ import Reflux from 'reflux';
 import _ from 'lodash';
 import * as Sentry from '@sentry/browser';
 
-const validHookNames = new Set([
+const __HookNames = [
```

Comment:
Does the `__` do anything? or is it a visual indicator of "don't use this"?

Reply:
@markstory there's nothing special. I just couldn't come up with a better variable name 😄

Reply:
Would prefer a diff name since we rarely use `__` in javascript. Maybe just `allHooks` or something.